src/components/Executions/ExecutionFilters.tsx

```diff
@@ -57,21 +91,41 @@ export const ExecutionFilters: React.FC<{ filters: FilterState[] }> = ({
     filters
 }) => {
     const styles = useStyles();
+    const { profile } = React.useContext(APIContext);
```

Comment:
I made a suggestion below to move the hiding logic into the filter state. If you do that, we don't need user profile here at all.